platforms/vercel-with-redwood/.editorconfig

```diff
@@ -0,0 +1,10 @@
+# editorconfig.org
```

Comment:
we shouldn't need a per-project editorconfig

Reply:
I think this comes from redwood started project. I will remove it, if needed.